### PR TITLE
[W6.4h][W11-B2]Chan Jun Yuan

### PR DESCRIPTION
**Pull request by tinyjy94**
Add Printable Interface and callers
Modify methods to use getPrintableString
Create Junit test to check getPrintableString

### CODE REVIEW

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printable.java:3]: What does this interface do? Need header comments to explain that.

File: src/seedu/addressbook/ui/Formatter.java [hunk 2]
```diff
@@ -34,6 +35,17 @@ public String format(String... messages) {
         return sb.toString();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/Formatter.java:39]: What would be a better place to have this method?

File: test/java/seedu/addressbook/data/PrintableTest.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                            new UniqueTagList(new Tag("Person"), new Tag("Good")));
+    }
+
+    @Test
+    public void printable_testPrivate() {
+        final String privateDetails = "Name: Johan Phone: (private) 123456 Email: (private) johan@gmail.com Address: (private) 321 Ave ";
+        assertPrivatePrintableString(privateDetails);
+    }
+
+    @Test
```
Comment (okkhoy) [inline test/java/seedu/addressbook/data/PrintableTest.java:41]: :+1:  good job on updating the tests.

**Comment by okkhoy** [comment]
@tinyjy94 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/ui/Formatter.java (hunk 2 above)
Comment (tinyjy94) [reply src/seedu/addressbook/ui/Formatter.java:39]: I think ReadOnlyPerson might be a better place to house this method

**Comment by tinyjy94** [comment]
Noted with thanks!

**Pull request closed without merging**